rudder-agent/SOURCES/Makefile

```diff
@@ -285,7 +285,7 @@ initial-ncf: ./rudder-sources
 	rm -rf ./initial-ncf
 	cp -r ./rudder-sources/ncf/tree ./initial-ncf
 	# retreive the rudder specific ncf.conf
-	cp ./initial-promises/node-server/distributePolicy/ncf/rudder-ncf-conf ./initial-ncf/ncf.conf
+	cp ./initial-promises/distributePolicy/ncf/rudder-ncf-conf ./initial-ncf/ncf.conf
```

Comment:
See https://github.com/Normation/rudder-packages/pull/1008/files#diff-27eac6d98b5f081d0f5cd3cdd3e22c99R282 for explanation.